backend/src/api.ts

```diff
@@ -246,7 +246,10 @@ loginCheckedPost('/signIn', async (req, user) => signIn(req.body.id, user));
 loginCheckedPost('/signInAll', async (_, user) => allSignInForms(user));
 
 // Team Events
-loginCheckedPost('/createTeamEvent', async (req, user) => createTeamEvent(req.body, user));
+loginCheckedPost('/createTeamEvent', async (req, user) => {
+  await createTeamEvent(req.body, user);
+  return {};
+});
```

Comment:
returning the value of `createteamEvent` was causing a type error and the frontend doesn't even read the value of the response so just returning `{}` here instead